release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4781,4 +4781,34 @@ $ oc adm release info 4.12.59 --pullspecs
 [id="ocp-4-12-59-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-60"]
+=== RHSA-2024:4006 - {product-title} 4.12.60 bug fix and security update
+
+Issued: 2024-06-27
+
+{product-title} release 4.12.60, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4006[RHSA-2024:4006] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4008[RHSA-2024:4008] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.60 --pullspecs
+----
+
+[id="ocp-4-12-60-bug-fixes"]
+==== Bug fixes
+
+* Previously, for the `cluster:capacity_cpu_cores:sum` metric, nodes with the`infra` role but not `master` role were not assigned a value of `infra` for the `label_node_role_kubernetes_io` label. With this update, nodes with the `infra` role but not `master` role are now correctly labeled as `infra` for this metric. (link:https://issues.redhat.com/browse/OCPBUGS-35558[*OCPBUGS-35558*])
+
+* Previously, when an IngressController was configured with client SSL/TLS, but did not have the `clientca-configmap` finalizer, the Ingress Operator would try to add the finalizer without checking whether the IngressController was marked for deletion. Consequently, if an IngressController was configured with client SSL/TLS and was subsequently deleted, the Operator would correctly remove the finalizer. It would then repeatedly, and erroneously, try and fail to update the IngressController to add the finalizer back, resulting in error messages in the Operator's logs.
```

Comment:
```suggestion
* Previously, when an Ingress Controller was configured with client SSL/TLS, but did not have the `clientca-configmap` finalizer, the Ingress Operator would try to add the finalizer without checking whether the Ingress Controller was marked for deletion. Consequently, if an Ingress Controller was configured with client SSL/TLS and was subsequently deleted, the Operator would correctly remove the finalizer. It would then repeatedly try and fail to update the IngressController to add the finalizer back, resulting in error messages in the Operator's logs.
```
See the repo glossary entry on the [Ingress Controller](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress-controller). Also, I suggest cutting _erroneously_ as it is fluff.